db/tag.c

```diff
@@ -5174,6 +5174,11 @@ static int init_default_value(struct field *fld, int fldn, int loadstore)
             if (strncasecmp(typebuf, "CURRENT_TIMESTAMP", 17) == 0)
                 is_null = 1; /* use isnull flag for current timestamp since
                                 null=yes is used for dbstore null */
+        } else if(opttype == CLIENT_BYTEARRAY) {
+            opttype = CLIENT_CSTR;
+            if (strncasecmp(typebuf, "GUID", 4) == 0)
```

Comment:
we have used strcmp() (case sensitive) in macc_so.c. Should change this to use that as well? IIUC csc2 parsing is mostly case sensitive.

Reply:
Good point -- probably strcmp is fine.

Reply:
Done.